rabbitmq/meta/collectd.yml

```diff
@@ -1,14 +1,12 @@
+{%- from "rabbitmq/map.jinja" import server with context %}
 plugin:
- # rabbitmq_server_rabbitmq:
-  #  plugin: rabbitmq
-   # interval: 60
-    #template: rabbitmq/files/collectd_rabbitmq.conf
+  rabbitmq_info:
+    plugin: python
+    execution: local
+    template: collectd/files/collectd_check_local_endpoint.conf
+    username: {{ server.admin.username }}
+    password: {{ server.admin.password }}
```

Comment:
In general, shall we pass every parameter used by the template like you've done it here for username & password?